.github/workflows/docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
-sphinx >= 4.3.2, <6.2.0
+sphinx == 6.2.1
```

Comment:
```suggestion
sphinx ~= 6.2
```
Do we need to pin to an exact version?

Reply:
Not really. I think our theme has to be <7 so I could change it to >= 4.3.2 <7 if preferred.

Reply:
No harm in requiring >= 6.2 though I think.

Reply:
done